Add Propel Data (#52)
data-engineering-community/data-engineering-wiki: Tools/Propel.md

---
Aliases: [Propel]
Tags: []
publish: true
---
[Propel](https://propeldata.com) is an Analytics Backend as a Service platform. Essentially a serverless alternative to Clickhouse or Elasticsearch.

## Propel Official Documentation

https://www.propeldata.com/docs

## Propel Advantages

> - A managed serverless service
> - Low latency, typically in milliseconds
> - Built-in timezone handling
> - Semantic layer for data modeling with GraphQL APIs

Reviewer: ```suggestion
- A managed serverless service
- Ingest data and speed up queries from any data source.  Low latency, typically in milliseconds
- Build at any layer of the stack: SQL, APIs, or React UI Components.
- Semantic layer for data modeling with GraphQL APIs
```

> ## Propel Disadvantages
> 
> - Learning curve due to the novel nature of the service

Reviewer: ```suggestion
## Use cases

- Embedded analytics: build analytics products.
- Data APIs: Instant low-latency APIs over any data source.
- Data Sharing: Give your customers their data.
- Self-service custom reporting: Empower your customers to build their own reports.
- ```

---
Aliases: [Propel]
Tags: []
publish: true
---
[Propel](https://propeldata.com) is an Analytics Backend as a Service platform. Essentially a serverless alternative to Clickhouse or Elasticsearch.

## Propel Official Documentation

https://www.propeldata.com/docs

## Propel Advantages

- A managed serverless service.
- Ingest data and speed up queries from any data source.
- Low latency, typically in milliseconds.
- Build at any layer of the stack: SQL, APIs, or React UI Components.
- Semantic layer for data modeling with GraphQL APIs.

## Use cases

- Embedded analytics: build analytics products.
- Data APIs: Instant low-latency APIs over any data source.
- Data Sharing: Give your customers their data.
- Self-service custom reporting: Empower your customers to build their own reports.

%% wiki footer: Please don't edit anything below this line %%

## This note in GitHub

> <span class="git-footer">[Edit In GitHub](https://github.dev/data-engineering-community/data-engineering-wiki/blob/main/Tools/Databricks.md "git-hub-edit-note") | [Copy this note](https://raw.githubusercontent.com/data-engineering-community/data-engineering-wiki/main/Tools/Databricks.md "git-hub-copy-note")</span>

Reviewer 2: ```suggestion
<span class="git-footer">[Edit In GitHub](https://github.dev/data-engineering-community/data-engineering-wiki/blob/main/Tools/Propel.md "git-hub-edit-note") | [Copy this note](https://raw.githubusercontent.com/data-engineering-community/data-engineering-wiki/main/Tools/Propel.md "git-hub-copy-note")</span>
```

<span class="git-footer">Was this page helpful?

> [👍](https://tally.so/r/mOaxjk?rating=Yes&url=https://dataengineering.wiki/Tools/Databricks) or [👎](https://tally.so/r/mOaxjk?rating=No&url=https://dataengineering.wiki/Tools/Databricks)</span>

Reviewer 2: ```suggestion
[👍](https://tally.so/r/mOaxjk?rating=Yes&url=https://dataengineering.wiki/Tools/Propel) or [👎](https://tally.so/r/mOaxjk?rating=No&url=https://dataengineering.wiki/Tools/Propel)</span>
```
